content/tutorials/manage-the-rippled-server/cluster-rippled-servers.md

```diff
@@ -1,22 +1,87 @@
 # Cluster rippled Servers
 
-If you are running multiple `rippled` servers in a single datacenter, you can configure those servers into a cluster to maximize efficiency. Running your `rippled` servers in a cluster provides the following benefits:
+If you run multiple `rippled` servers in the same data center, you can configure them in a [cluster](clustering.html) to maximize efficiency. To configure clustering:
 
-* Clustered `rippled` servers share the work of cryptography. If one server has verified the authenticity of a message, the other servers in the cluster trust it and do not re-verify.
-* Clustered servers share information about peers and API clients that are misbehaving or abusing the network. This makes it harder to attack all servers of the cluster at once.
-* Clustered servers always propagate transactions throughout the cluster, even if the transaction does not meet the current load-based transaction fee on some of them.
+1. For each of your servers, note the IP address of the server.
```

Comment:
Would it be worth moving the benefits of clustering points to the concepts doc? Those bullet points seem like good information to provide.

Reply:
They are already in the concept article (lines 5-7, just above the excerpt GitHub shows)